content/en/includes/task-tutorial-prereqs-node-upgrade.md

```diff
@@ -0,0 +1,3 @@
+You need to have shell access to all the nodes, and the kubectl command-line tool must
+be configured to communicate with your cluster. It is recommended to run this tutorial 
+on a cluster with at least two nodes that are not acting as control plane hosts.
```

Comment:
This is OK; however, I can see some nits.

- the pages we're including it in aren't tutorials
- we don't mention that you need to have a cluster that includes at least one node that needs upgrading
  - actually, you don't need to have shell access to the control plane nodes (AIUI)
- I'd write `kubectl` in backticks

Reply:
Thanks for reviewing @sftim. Should I reopen this PR and apply your suggestions? Why did you approve it?

Reply:
This change was good enough to merge, and seemed worth making.

If (and only if) you've capacity to make a further change, you can - it would need a new PR. It's also fine to remember this feedback, but leave things be in terms of these two pages specifically.

Reply:
Thank you.